### PR TITLE
Test fixes added

### PR DESCRIPTION
**Pull request by kumarks1122**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: data-products/src/test/scala/org/sunbird/analytics/audit/TestScoreMetricMigrationJob.scala [hunk 1]
```diff
@@ -43,25 +43,18 @@ class TestScoreMetricMigrationJob extends BaseSpec with MockFactory {
...
 
-    aggDetail("max_score") should be(10.0)
-    aggDetail("score") should be(10.0)
-    aggDetail("type") should be(jobConfig.modelParams.get.get("metricsType").get.toString)
-    aggDetail("attempt_id") should be("attempat-001")
-    //aggDetail("content_id") should be("do_112876961957437440110")
-    aggDetail("attempted_on") should be(1634810023)
-
-    result.head.get(2).asInstanceOf[Seq[String]](1) should be("""{"max_score":10.0,"score":10.0,"type":"attempt_metrics","attempt_id":"attempat-001","content_id":"do_112876961957437440179"}""")
+    result.head.get(2).asInstanceOf[Seq[String]] should contain allElementsOf List("""{"max_score":10.0,"score":10.0,"type":"attempt_metrics","attempt_id":"attempat-001","content_id":"do_112876961957437440110","attempted_on":1634810023}""", """{"max_score":10.0,"score":10.0,"type":"attempt_metrics","attempt_id":"attempat-001","content_id":"do_112876961957437440179"}""")
```
Comment (anandp504) [inline data-products/src/test/scala/org/sunbird/analytics/audit/TestScoreMetricMigrationJob.scala]: Please deserialize to a case class and then perform the assertions. This won't work if the keys in the JSON string is in a different order. JSON is not an ordered list of key, value pairs.

**Review by anandp504 — CHANGES_REQUESTED** [review]
Please implement the review comment changes.

File: data-products/src/test/scala/org/sunbird/analytics/audit/TestScoreMetricMigrationJob.scala (hunk 1 above)
Comment (kumarks1122) [reply data-products/src/test/scala/org/sunbird/analytics/audit/TestScoreMetricMigrationJob.scala]: @anandp504 : Sure

**Review by anandp504 — APPROVED** [review]
Approved :+1:

**Pull request merged**